crates/bevy_light/src/directional_light.rs

```diff
@@ -182,6 +183,8 @@ pub struct DirectionalLightTexture {
 pub struct DirectionalLightShadowMap {
     // The width and height of each cascade.
     ///
+    /// Must be a power of two to avoid unstable cascade positioning.
+    ///
```

Comment:
Instead of documenting here, it may be better to make this a private field, and enforce the power of two requirement at construction.

Reply:
This was considered: https://discord.com/channels/691052431525675048/743663924229963868/1400667437829066754